APIClient/Base.lproj/Main.storyboard

```diff
@@ -1,44 +1,77 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10117" systemVersion="15F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="E10-YL-lxp">
+<<<<<<< HEAD
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10116" systemVersion="15E65" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="E10-YL-lxp">
```

Comment:
Bei dem Merge sind hier offenbar Konflikte aufgetreten, die musst du noch beheben. Einfach die `.storyboard` Datei in einem Texteditor öffnen und bei diesen Konflikt-Markierungen den oberen oder unteren Teil löschen, je nach dem was du behalten willst.